packages/node_modules/pouchdb-core/src/changes.js

```diff
@@ -169,6 +170,11 @@ Changes.prototype.doChanges = function (opts) {
     opts.filter = '_view';
   }
 
+  /* istanbul ignore if */
+  if (opts.selector && !opts.filter) {
```

Comment:
Does this mean that a _view filter will take precedence over a mango selector? Is this how CouchDB treats it?